odoo/addons/base/views/ir_model_views.xml

```diff
@@ -387,6 +399,54 @@
         </record>
         <menuitem action="action_model_fields" id="ir_model_model_fields" parent="base.next_id_9"/>
 
+        <!-- ir model fields selection -->
+        <record id="view_model_fields_selection_form" model="ir.ui.view">
+            <field name="model">ir.model.fields.selection</field>
+            <field name="arch" type="xml">
+                <form string="Fields" duplicate="false">
+                    <sheet>
+                        <group>
+                            <field name="field_id"/>
+                            <field name="value" groups="base.group_no_one"/>
+                            <field name="name"/>
+                            <field name="sequence" groups="base.group_no_one"/>
+                        </group>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+        <record id="view_model_fields_selection_tree" model="ir.ui.view">
+            <field name="model">ir.model.fields.selection</field>
+            <field name="arch" type="xml">
+                <tree string="Fields Selections">
```

Comment:
The `string` attribute [is deprecated](https://github.com/odoo/odoo/blob/6191285c6a425759c017157a7357d02db298b519/odoo/addons/base/rng/tree_view.rng#L32)  (no longer displayed) since version 8.0.
```suggestion
                <tree>
```